lib/plugins/rest.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
+var Instance = require("../services/instanceAdapter.js");
```

Comment:
I think we should call this `Instances` or `InstanceAdapter`. `Instance` already means a model class.